photutils/isophote/harmonics.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import (absolute_import, division, print_function, unicode_literals)
+
+import numpy as np
+from scipy.optimize import leastsq
```

Comment:
@ibusko - scipy is currently an optional dependency of photutils, so this will fail the tests. 

In other modules (e.g segmentation) this is avoided by importing scipy in the function/class rather on modul level. Corresponding tests then are skipped when no scipy install is available.